sentry-core/src/main/java/io/sentry/core/Session.java

```diff
@@ -13,7 +13,8 @@
   public enum State {
     Ok,
     Exited,
-    Crashed
+    Crashed,
+    Abnormal // not used by mobile SDKs
```

Comment:
L: Abnormal is still used on sentry-cocoa so current comment is a bit misleading. Maybe:

```suggestion
    Abnormal // not currently used in this SDK.
```